styles/styles.scss

```diff
@@ -33,3 +33,7 @@
 @import 'ej2-dropdowns/styles/multi-select/all.scss';
 @import 'ej2-popups/styles/tooltip/fabric-definition.scss';
 @import 'ej2-popups/styles/tooltip/all.scss';
+@import 'ej2-base/styles/fabric.scss';
+@import 'ej2-navigations/styles/fabric.scss';
```

Comment:
Can we simplify the these imports to more specific packages?

Reply:
Import Styles Changes Done.